apps/mull-ui-e2e/src/integration/create-event.spec.ts

```diff
@@ -1,85 +1,98 @@
-  });
-
-  it('should enter the end time', () => {
-    cy.get(':nth-child(6) > [data-testid=custom-time-input]')
-      .type('15:20')
-      .should('have.value', '15:20');
-  });
-
-  it('should type into the event title input', () => {
-    cy.get(':nth-child(7) > [data-testid=custom-text-input]')
-      .type('test title')
-      .should('have.value', 'test title');
-  });
-
-  it('should select today as the start and end date', () => {
-    cy.get('.-today').click().should('have.text', new Date().getDate().toString());
-    cy.get('.-today').click().should('have.text', new Date().getDate().toString());
-  });
-
-  it('should type into the event description input', () => {
-    cy.get(':nth-child(8) > [data-testid=custom-text-input]')
-      .type('test description')
-      .should('have.value', 'test description');
-  });
-  it('should type into the event location input', () => {
-    cy.get(':nth-child(9) > [data-testid=custom-text-input]')
-      .type('test location')
-      .should('have.value', 'test location');
-  });
-
-  it('should change restriction open', () => {
-    cy.get('[data-testid=pill-id-1]')
-      .click()
-      .should('have.css', 'background-color', 'rgb(39, 176, 154)');
-    cy.get('[data-testid=pill-id-2]')
-      .click()
-      .should('have.css', 'background-color', 'rgb(39, 176, 154)');
-  });
-
-  it('should show errors when fields are not completed', () => {
-    cy.get('.create-event-button').click();
-    cy.get(':nth-child(5) > .error-message').should('have.text', 'Start Time is required.');
-    cy.get(':nth-child(6) > .error-message').should('have.text', 'End Time is required.');
-    cy.get(':nth-child(7) > .error-message').should('have.text', 'Event Title is required.');
-    cy.get(':nth-child(8) > .error-message').should('have.text', 'Event Description is required.');
-    cy.get(':nth-child(9) > .error-message').should('have.text', 'Event Location is required.');
-  });
-
-  it('should show a successful submission message', () => {
-    cy.get(':nth-child(5) > [data-testid=custom-time-input]').type('11:20');
-
-    cy.get(':nth-child(6) > [data-testid=custom-time-input]').type('15:20');
-
-    cy.get(':nth-child(7) > [data-testid=custom-text-input]').type('test title');
-
-    cy.get('.-today').click();
-    cy.get('.-today').click();
-
-    cy.get(':nth-child(8) > [data-testid=custom-text-input]').type('test description');
-
-    cy.get(':nth-child(9) > [data-testid=custom-text-input]').type('test location');
-
-    cy.get('[data-testid=pill-id-1]').click();
-    cy.get('.create-event-button').click();
-
-    cy.get('.Toastify__toast.Toastify__toast--success', { timeout: 1000 }).should(
-      'have.css',
-      'background-color',
-      'rgb(39, 176, 154)'
-    );
+const frameSizes = [
+  [375, 812],
+  [1920, 1080],
+];
+
+frameSizes.forEach((size) => {
+  describe('US-1.1: Create Events', () => {
+    beforeEach(() => {
+      cy.viewport(size[0], size[1]);
```

Comment:
FYI: The changes I added are lines 3-6, 8, and 11. The rest of the lines are changed due to tabs.